tailwind.config.js

```diff
@@ -1,129 +1,239 @@
+    display: '"Titan One", sans-serif',
+  },
+  nyias: {
+    body: '"Montserrat", sans-serif',
+    display: '"Titan One", sans-serif',
+  },
+  dragons: {
+    body: '"Montserrat", sans-serif',
+    display: '"Titan One", sans-serif',
+  },
+  grey: {
+    body: '"Montserrat", sans-serif',
+    display: '"Titan One", sans-serif',
+  },
+  forest: {
+    body: '"Tenor Sans", sans-serif',
     display: '"Merienda", cursive',
   },
   racer: {
-		body: "Montserrat",
-    display: "Anton",
+    body: '"Montserrat", sans-serif',
+    display: '"Anton", sans-serif',
+  },
+  retroDiner: {
+    body: '"Montserrat", sans-serif',
+    display: '"Roboto Slab", serif',
   },
-  "retro-diner": {
-		body: "Montserrat",
-    display: "Roboto Slab",
+  duke: {
+    body: '"Open Sans", sans-serif',
+    display: '"EB Garamond", serif',
+  },
+  raven: {
+    body: '"Montserrat", sans-serif',
+    display: '"Anton", sans-serif',
+  },
+  slateGreen: {
+    body: '"Mulish", sans-serif',
+    display: '"Chakra Petch", sans-serif',
+  },
+  roseAmber: {
+    body: '"Quicksand", sans-serif',
+    display: '"Satisfy", cursive',
+  },
+  violetSky: {
+    body: '"Montserrat", sans-serif',
+    display: '"Delius", cursive',
   },
 };
 
 const tailwindColors = {
-	patchWRK: {
-		main: "white",
-		tertiary: "theme('colors.neutral.50')",
-		inactive: "theme('colors.blue.200')",
-		secondary: "theme('colors.slate.400')",
-		"accent-light": "theme('colors.slate.500')",
-		"trim-light": "theme('colors.slate.500')",
-		"accent-dark": "theme('colors.blue.700')",
-		primary: "theme('colors.blue.900')",
-		"trim-dark": "theme('colors.blue.900')",
-		info: "#0a0a0a",
-		success: "#22C55E",
-		error: "#F97316",
-		warning: "#DC2626",
-	},
-	nyias: {
-		main: "white",
-		tertiary: "theme('colors.neutral.50')",
-		inactive: "theme('colors.neutral.200')",
-		secondary: "theme('colors.red.400')",
-		"accent-light": "theme('colors.red.500')",
-		"trim-light": "theme('colors.red.500')",
-		"accent-dark": "theme('colors.neutral.700')",
-		primary: "theme('colors.neutral.900')",
-		"trim-dark": "theme('colors.neutral.900')",
-		info: "#0a0a0a",
-		success: "#22C55E",
-		error: "#F97316",
-		warning: "#DC2626",
-	},
-	dragons: {
-		main: "white",
-		tertiary: "theme('colors.neutral.50')",
-		inactive: "theme('colors.purple.200')",
-		secondary: "theme('colors.yellow.400')",
-		"accent-light": "theme('colors.yellow.500')",
-		"trim-light": "theme('colors.yellow.500')",
-		"accent-dark": "theme('colors.purple.700')",
-		primary: "theme('colors.purple.900')",
-		"trim-dark": "theme('colors.purple.900')",
-		info: "#0a0a0a",
-		success: "#22C55E",
-		error: "#F97316",
-		warning: "#DC2626",
-	},
+  patchWrk: {
+    main: "#ffffff",
+    tertiary: "#f9fafb",
+    inactive: "#bfdbfe",
+    secondary: "#94a3b8",
+    accentLight: "#64748b",
+    trimLight: "#64748b",
+    accentDark: "#1d4ed8",
+    primary: "#1e3a8a",
+    trimDark: "#1e3a8a",
+    info: "#0a0a0a",
```

Comment:
@eprisr Please add,
infoLight: "#ffffff",
in future code...
infoLight is a newly Named Schema Color in the stack